### PR TITLE
Document how to set up B2B module and Payment on account

### PR DESCRIPTION
**Pull request by dpobel**
2 new pages:

* https://deploy-preview-259--heuristic-almeida-1a1f35.netlify.app/docs/magento2/b2b.html
* https://deploy-preview-259--heuristic-almeida-1a1f35.netlify.app/docs/advanced/payments/payment-on-account.html

### CODE REVIEW

**Review by abailleulfc — APPROVED** [review]

File: source/docs/advanced/payments/payment-on-account.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+### Register the Payment on account payment component
+
+1. Override the file that lets you register additional payments components in Front-Commerce
+  ```bash
+mkdir -p my-module/web/theme/modules/Checkout/Payment/AdditionalPaymentInformation/
+cp -u node_modules/front-commerce/src/web/theme/modules/Checkout/Payment/AdditionalPaymentInformation/getAdditionalDataComponent.js my-module/web/theme/modules/Checkout/Payment/AdditionalPaymentInformation/getAdditionalDataComponent.js
+  ```
+  1. Register the component `PaymentOnAccount` to used for `companycredit` payments in `getAdditionalDataComponent.js`
+  ```diff
++import PaymentOnAccount from "./PaymentOnAccount";
```
Comment (mo-fc) [inline source/docs/advanced/payments/payment-on-account.md:31]: where is `./PaymentOnAccount` it is not clear.
maybe add absolute path like `theme/...`

Comment (dpobel) [reply source/docs/advanced/payments/payment-on-account.md]: agreed and fixed (it's modeled after the other payment method documentation pages)

File: source/docs/advanced/payments/payment-on-account.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+title: Payment on account
+---
+
+Payment on account is a payment method for B2B context that allows companies to make purchases up to the credit limit that is specified in their profile.
+
+<blockquote class="feature--new">
+_Since version 2.11.0_
+</blockquote>
+
+## Enable Payment on account
```
Comment (real34) [inline source/docs/advanced/payments/payment-on-account.md:12]: For consistency with other payment methods, I think we should add a new title level with the Platform.

```
## Magento2 B2B

<blockquote class="feature--new">
_Since version 2.11.0_
</blockquote>

### Enable Payment on account
```

File: source/docs/advanced/payments/payment-on-account.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
+---
+id: payment-on-account
+title: Payment on account
+---
+
+Payment on account is a payment method for B2B context that allows companies to make purchases up to the credit limit that is specified in their profile.
```
Comment (real34) [inline source/docs/advanced/payments/payment-on-account.md:6]:
```suggestion
Payment on account is a payment method for B2B contexts. It allows companies to place orders using the credit limit specified in their profile.
```

File: source/docs/magento2/advanced.md [hunk 4]
```diff
@@ -3,7 +3,7 @@ id: magento2-advanced
 title: Advanced
 ---
 
-In this section, you will learn some advanced usages of the Magento 1 integration.
+In this section, you will learn some advanced usages of the Magento 2 integration.
```
Comment (real34) [inline source/docs/magento2/advanced.md:6]: :+1:

File: source/docs/magento2/b2b.md [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+
+<blockquote class="feature--new">
+_Since version 2.11.0_
+</blockquote>
+
+Front-Commerce supports several Magento2 B2B features:
+
+* Display details of the Company a customer belongs to
+* Company users handling (list, create, modify and deactivate company users)
+* _Payment on account_ payment method
```
Comment (real34) [inline source/docs/magento2/b2b.md:14]:
```suggestion
* [_Payment on account_](/docs/advanced/payments/payment-on-account.html) payment method
```

File: source/docs/magento2/b2b.md [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+// src/web/theme/main.scss
+@import "~front-commerce/theme-chocolatine/web/theme/main";
+/*
+if you use the default theme as a base theme, you should use:
+@import "~front-commerce/src/web/theme/main.scss";
+*/
+@import "~theme/b2b";
+  ```
+
+And that's it. After having restarted Front-Commerce, the B2B module should be enabled and integrated into your project. If you login as a company user and then go to the user account dashboard, you should be able to access the _Company details_, _Company credit_ and _Company users_ pages.
```
Comment (real34) [inline source/docs/magento2/b2b.md:68]: Let's keep it vague to make it future proof :)
```suggestion
And that's it. After having restarted Front-Commerce, the B2B module should be enabled and integrated into your project. If you login as a company user and then go to the user account dashboard, you should see new B2B specific menu entries (company management, credit history…).
```

**Review by real34 — APPROVED** [review]

**Pull request merged**